web-animations-1/Overview.bs

```diff
@@ -4699,6 +4704,18 @@ interface KeyframeEffect : AnimationEffect {
 
     1.  Set the <a>target element</a> of <var>effect</var> to <var>target</var>.
 
+    1.  Set the [=target pseudo-selector=] to the result corresponding to the first
```

Comment:
Editorial nits:

- Above line uses `<a>target element</a>`, this uses `[=target pseudo selector=]`.
- Above line mentions `effect`, this omits it.

Reply:
done